inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/utils/ViewUtil.kt

```diff
@@ -74,6 +74,23 @@ internal object ViewUtil {
         }
     }
 
+    fun getEdgePosition(width: Int, height: Int, topPos: Pair<Int, Int>): Pair<Int?, Int?> {
+        val windowW = Resources.getSystem().displayMetrics.widthPixels
+        val windowH = Resources.getSystem().displayMetrics.heightPixels
```

Comment:
For next version improvement, please consider window height with navigation bar

Reply:
thank you.
added in future improvement list.